include/openenclave/internal/sgx/sgxproperties.h

```diff
@@ -32,7 +32,7 @@ OE_INLINE bool oe_sgx_is_valid_num_tcs(uint64_t x)
 OE_INLINE bool oe_sgx_is_valid_attributes(uint64_t x)
```

Comment:
I had a similar comment - "oe_sgx_validate_enclave_properties() should check family_id and ext_product_id too"

Reply:
family_id or ext_product_id can be any value. Nothing is considered invalid. The check is unnecessary.

Reply:
@jianlia @yentsanglee 

I agree that in terms of the structure provide, any fixed 16-byte value of `family_id` and `extended_product_id` would be considered valid. The remaining concern still needs to be addressed:

> Specifically, this might need to cross-validate the attributes with the presence of the `family_id` or `ext_product_id` values, see comment in oesign.

Effectively, the check in oesign for `(attributes & `OE_SGX_FLAGS_KSS`) == 0` means that `family_id` and `extended_product_id`  must also be zero arrays needs to be enforced as inconsistent and rejected by the loader. That bit of logic is what needs to be reusable as a separate function.

Reply:
Add  oe_sgx_is_unset_uuid function per comment.